.jenkins/jenkins_buildbot_python2_debug.sh

```diff
@@ -9,6 +9,40 @@ export PATH=/usr/local/cuda/bin:$PATH
 export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
 export LIBRARY_PATH=/usr/local/cuda/lib64:$LIBRARY_PATH
 
+GPUARRAY_CONFIG="Release"
```

Comment:
This is copy-pasted from the Theano GPU buildbot.
I think compiling libgpuarray in release is what we want, as we do not need slowness and debugging symbols.